tournament/on.cfg

```diff
@@ -16,7 +16,31 @@ sv_countdown_unpause "10"
 # sv_countdown_round_start, 0, -1, 1000, "Number of seconds to freeze the game in a countdown before match starts (0 enables only for survival gamemodes, -1 disables)"
 sv_countdown_round_start "0"
 
-sv_timelimit "0"
+sv_vote_checkboxes 0
+
+sv_destroy_bullets_on_death 0
+sv_destroy_lasers_on_death 0
+
+
+sv_vote_checkboxes 0
+
+sv_countdown_unpause "10"
+sv_countdown_round_start "0"
+sv_show_settings_motd "0"
+sv_force_ready_all "5"
+
+sv_only_hook_kills 0
+sv_kill_hook 0
+
+sv_vote_checkboxes 1
+sv_vote_checkboxes 1
+
+sv_only_hook_kills 0
+sv_kill_hook 0
```

Comment:
sv_kill_hook is already set to 0 in line 33 and 55. Make sure to check all configs for duplicated before adding them. It will be a maintenance mess later. If all configs are shadowed by duplicates.